lib/spi/cluster-contracts.ts

```diff
@@ -1,6 +1,6 @@
 import { ClusterInfo } from '.';
-import { Construct } from '@aws-cdk/core';
-import { IVpc } from '@aws-cdk/aws-ec2';
+import { Construct } from "constructs";
+import { IVpc } from 'aws-cdk-lib/aws-ec2';
```

Comment:
mixed quotes

Reply:
it is a global replace you have to choose something.